docs/source/Plugins.rst

```diff
@@ -88,7 +88,7 @@ We have briefly discussed the making of Avocado plugins. We recommend
 the `Stevedore documentation`_ and also a look at the
 :mod:`avocado.plugins.base` module for the various plugin interface definitions.
 
-Some plugins examples are available in the `Avocado source tree_`, under ``examples/plugins``.
+Some plugins examples are available in the `Avocado source tree`_, under ``examples/plugins``.
```

Comment:
No problem, it was really tricky to find (as my eyes tend to reorder characters as they think they should be. So I had to really REALLY debug it to see why most of the link works and one does not... These are the worst mistakes :smile: